src/Radicle/Internal/Effects.hs

```diff
@@ -79,17 +81,41 @@ replBindings = e { bindingsPrimFns = bindingsPrimFns e <> replPrimFns
       pfs = replPrimFns
 
 replPrimFns :: forall m. ReplM m => PrimFns m
-replPrimFns = PrimFns . Map.fromList $ first unsafeToIdent <$>
+replPrimFns = fromList $ Doc.noDocs $ first unsafeToIdent <$>
     [ ("print!", \case
         [x] -> do
             putStrS (renderPrettyDef x)
             pure nil
         xs  -> throwErrorHere $ WrongNumberOfArgs "print!" 1 (length xs))
 
+    , ( "doc!"
```

Comment:
This isn't actually impure, right? Is the idea that documentation raises questions about referential transparency?

Reply:
Well, it's as impure as `print!`.

Reply:
(It print's a formatted text to output, rather than return a string.)